### PR TITLE
Overmap Wormholes

### PR DESCRIPTION
**Pull request by Geevies**
* Added wormholes to the overmap.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/overmap/triggers/triggers.dm [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	paired_wormhole = WEAKREF(sibling_wormhole)
+	sibling_wormhole.paired_wormhole = WEAKREF(src)
+
+/obj/effect/overmap/trigger/wormhole/handle_crossed(var/obj/effect/overmap/visitable/ship/ship)
+	var/obj/effect/overmap/trigger/wormhole/sibling_wormhole = paired_wormhole.resolve()
+	ship.position = list(0, 0)
+	ship.forceMove(get_turf(sibling_wormhole))
+	ship.update_icon()
+
+	for(var/mob/living/carbon/human/human as anything in human_mob_list)
```
Comment (NonQueueingMatt) [inline code/modules/overmap/triggers/triggers.dm:63]:
```suggestion
	for(var/mob/living/carbon/human/human in human_mob_list)
```

**Review by NonQueueingMatt — DISMISSED** [review]

**Comment by Geevies** [comment]
!review

**Comment by NonQueueingMatt** [comment]
Unwanted by loremasters.

**Pull request closed without merging**